drivers/sensors/cirque_pinnacle.h

```diff
@@ -43,7 +34,10 @@ void            cirque_pinnacle_set_scale(uint16_t scale);
 #ifndef CIRQUE_PINNACLE_Y_RANGE
 #    define CIRQUE_PINNACLE_Y_RANGE (CIRQUE_PINNACLE_Y_UPPER - CIRQUE_PINNACLE_Y_LOWER)
 #endif
-
+#if !defined(POINTING_DEVICE_TASK_THROTTLE_MS) || POINTING_DEVICE_TASK_THROTTLE_MS < 10
+#    undef POINTING_DEVICE_TASK_THROTTLE_MS
+#    define POINTING_DEVICE_TASK_THROTTLE_MS 10 // Cirque Pinnacle at most will have fresh data every 10ms
+#endif
```

Comment:
I'm not a fan of overriding use selection to force a higher throttle interval.
While throughput will be similar, having 10ms interval will result in higher latency. Also could miss a window every now and then as the Pinnacle timer and keyboard timer get slightly out of sync.

```suggestion
#ifndef POINTING_DEVICE_TASK_THROTTLE_MS
#    define POINTING_DEVICE_TASK_THROTTLE_MS 10 // Cirque Pinnacle will have fresh data every 10ms by default
#endif
```